webui/react/src/components/UPlotChart.tsx

```diff
@@ -25,6 +30,7 @@ const UPlotChart: React.FC<Props> = forwardRef((
 ) => {
   const [ chart, setChart ] = useState<uPlot>();
   const chartDivRef = useRef<HTMLDivElement>(null);
+  const isZoomed = useRef<boolean>(false);
```

Comment:
Is there a reason this is a ref instead of just useState?

Reply:
Yes, to avoid triggering re-renders.

Setting a useState causes a re-render (which might be fine) but is not needed in a case like this.

useRef doesn't :)